src/redis-cli.c

```diff
@@ -213,6 +213,7 @@ static struct config {
     int shutdown;
     int monitor_mode;
     int pubsub_mode;
+    int block_state_aborted;
```

Comment:
I think a comment will be useful, and i also suggest a rename:
```suggestion
    int blocking_state_aborted; /* Used to abort monitor_mode and pubsub_mode. */
```